Fitbit.Portable/Models/Pagination.cs

```diff
@@ -7,7 +7,7 @@ namespace Fitbit.Api.Portable.Models
 {
     public class Pagination
         {
-            public string BeforeDate { get; set; }
+            public DateTime BeforeDate { get; set; }
```

Comment:
Do they always serialize / parse into `DateTime`s? As long as they do, this is a good change 👍

Reply:
It seems like we are pretty consistent on the library level of parsing into a DateTime object. And after taking a peek at the web api reference it seems to be as a date time object as well.

Reply:
Cool!